daemon/SecretsImpl/secretsrequestprocessor.cpp

```diff
@@ -6133,14 +6197,19 @@ Daemon::ApiImpl::RequestProcessor::userInputInteractionCompleted(
                         returnResult = Result(Result::UnknownError,
                                               QLatin1String("Internal error: incorrect parameter count!"));
                     } else {
+                        Secret::Identifier identifier = pr.parameters.takeFirst().value<Secret::Identifier>();
+                        Sailfish::Crypto::CryptoManager::Operation operation = pr.parameters.takeFirst().value<Sailfish::Crypto::CryptoManager::Operation>();
+                        QString cryptoPluginName = pr.parameters.takeFirst().value<QString>();
+                        CollectionMetadata collectionMetadata = pr.parameters.takeFirst().value<CollectionMetadata>();
+                        SecretManager::UserInteractionMode userInteractionMode = static_cast<SecretManager::UserInteractionMode>(pr.parameters.takeFirst().value<int>());
```

Comment:
Order of taking out parameters has changed. Intentional?

Reply:
Good question, let's see...

Reply:
Yes, good catch. Still seems that there are places that push the same request with only 2 arguments, but not really relevant for this.